core/transcoder.go

```diff
@@ -189,7 +189,18 @@ func resToTranscodeData(res *ffmpeg.TranscodeResults, opts []ffmpeg.TranscodeOpt
 				glog.Error("Cannot read transcoded output for ", oname)
 				return nil, err
 			}
-			segments = append(segments, &TranscodedSegmentData{Data: o, Pixels: res.Encoded[i].Pixels})
+			// Extract perceptual hash if calculated
+			var s []byte = nil
+			if opts[i].CalcSign {
+				sigfile := oname + ".bin"
+				s, err = ioutil.ReadFile(sigfile)
+				if err != nil {
+					glog.Error("Cannot read perceptual hash at ", sigfile)
+					return nil, err
+				}
+				os.Remove(sigfile)
```

Comment:
[os.Remove](https://pkg.go.dev/os#Remove) returns an error. Let's handle it? Could just log the error and continue.